client/Dockerfile

```diff
@@ -12,21 +12,16 @@ ENV AUTH_CREDENTIALS $AUTH_CREDENTIALS
 
 WORKDIR /app
 
-
 COPY package.json pnpm-workspace.yaml pnpm-lock.yaml tsconfig.json ./
-COPY client/package.json client/tsconfig.json ./client/
-COPY shared/package.json shared/tsconfig.json ./shared/
-
-RUN corepack enable pnpm
-
 COPY client ./client
 COPY shared ./shared
 
+RUN corepack enable pnpm
+RUN pnpm shared:deps
```

Comment:
I understand the fix and why now it works, but it feels weird as fuck to see that pnpm does not handle this dependency graph on its own (when apparently in 9.4.x did)

Let's have a quick chat about it later cc @alepefe-vizz

Reply:
We are resolving this by running `pnpm install` and letting `pnpm` handle the dependencies to install.